cmd/canaryd/README.md

```diff
@@ -68,6 +70,20 @@ $ MANIFEST_URL=http://www.canary.io/manifest.json DEFAULT_SAMPLE_INTERVAL=4 RAMP
 ^C
 ```
 
+## Manifest reloading
+
+`canaryd` supports manifest reloading via two means:
+
+- SIGHUP - Canaryd queries the defined MANIFEST_URL and reloads for changes in the defined targets.
+- Automatic reloading - `canaryd` will poll the MANIFEST_URL for changes via the interval defined in the AUTO_RELOAD_INTERVAL environment variable. This variable is a floating point value for the number of seconds that canaryd should poll for manifest changes, with 1, 15.0 and 0.25 all being valid. 
```

Comment:
it's unclear why we'd support anything smaller than a second for the reload interval.